libexec/alluxio-config.sh

```diff
@@ -92,14 +89,13 @@ ALLUXIO_SERVER_CLASSPATH="${ALLUXIO_CONF_DIR}/:${ALLUXIO_CLASSPATH}:${ALLUXIO_AS
 ## Start reading site-properties to set certain variables.
 ####################################################################################################
 function getConf {
-  CLASS="alluxio.cli.GetConf"
-  ALLUXIO_SHELL_JAVA_OPTS+=" -Dalluxio.logger.type=Console"
-  "${JAVA}" -cp ${ALLUXIO_CLIENT_CLASSPATH} ${ALLUXIO_SHELL_JAVA_OPTS} ${CLASS} "$1"
+  local jclass="alluxio.cli.GetConf"
+  "${JAVA}" -cp ${ALLUXIO_CLIENT_CLASSPATH} ${ALLUXIO_JAVA_OPTS} -Dalluxio.logger.type=Console ${jclass} "$1"
```

Comment:
That's because I want to make the line shorter, or maybe I can use \ at the end of the line.